web-marketplace/src/pages/Orders/Orders.tsx

```diff
@@ -0,0 +1,109 @@
+import { useNavigate } from 'react-router-dom';
+import { from } from '@apollo/client';
+import { Trans } from '@lingui/macro';
+import { useQuery } from '@tanstack/react-query';
+
+import { Flex } from 'web-components/src/components/box';
+import { Title } from 'web-components/src/components/typography/Title';
+import { cn } from 'web-components/src/utils/styles/cn';
+
+import { getGeocodingQuery } from 'lib/queries/react-query/mapbox/getGeocodingQuery/getGeocodingQuery';
+
+import { usePathSection } from 'pages/ProfileEdit/hooks/usePathSection';
+import { ProfileEditNav } from 'pages/ProfileEdit/ProfileEdit.Nav';
+import WithLoader from 'components/atoms/WithLoader';
+import { Order } from 'components/organisms/Order/Order';
+import { ORDER_STATUS } from 'components/organisms/Order/Order.constants';
+import {
+  blockchainDetails,
+  credits,
+  paymentInfo,
+  retirementInfo,
```

Comment:
I didn't translate them because they are mocks and we may end up creating them in a different way, perhaps inside the component with the real data I thought.

Reply:
yeah but the titles aren't going to change, they are not dynamic data, so we could put them directly in the components

Reply:
in the dashboard case, this is only static info that we extracted so it can be reusable (in the menu and on the dashboard)
Here it's a bit different, I just find a bit weird to mix up static info with dynamic content.